tests/unit/task_system_test.c

```diff
@@ -61,7 +61,8 @@ CTEST_DATA(task_system)
    // Config structs required, to exercise task subsystem
    io_config io_cfg;
 
-   uint8 num_bg_threads[NUM_TASK_TYPES];
+   uint8  num_bg_threads[NUM_TASK_TYPES];
```

Comment:
`uint64`

Reply:
Did not change this as that would percolate to the interface of many other functions that are taking `num_bg_threads[]` as an argument. (Also, this is existing code ... and will probably be reworked in future revs any ways.)